content/docs/command-reference/metrics/show.md

```diff
@@ -27,8 +27,7 @@ target can even be directories, so that DVC recursively shows all metrics files
 inside.
 
 > Note that targets don't necessarily have to be defined in `dvc.yaml`. For that
-> reason, this command doesn't require an existing DVC project to run in; It
-> works in any Git repo.
+> reason, this command doesn't require an existing DVC project to run in.
```

Comment:
Git repo is not necessary, removing this mention (only in `show`, because in diff we actually need it). Is removal ok, or should I explicitly state that for `show` Git is not necessary?

Reply:
Hmmm. I guess it's best to be explicit just in case:
```suggestion
> reason, this command doesn't require an existing DVC project or Git repo to run in.
```